adr/0013-suma-integration.md

```diff
@@ -0,0 +1,141 @@
+# 13. SUMA Integration
+
+Date: 2024-05-07
+
+## Status
+
+Accepted
+
+## Context
+
+Keeping SAP infrastructure up-to-date and secure is a crucial task for system administrators, allowing vulnerabilities reduction and overall system improvement in terms of features and performance.
```

Comment:
To be honest, this initial chapter looks a bit "actually" out of context.
As it doesn't talk about any technical challenge.
It talks about about a product new feature.

I think the context should talk about the challenge of communicating with a 3rd party component that doesn't have any reactiveness, and we are forced to poll data in periodical basis.